Kernel/Graphics/GraphicsManagement.cpp

```diff
@@ -230,4 +233,18 @@ bool GraphicsManagement::framebuffer_devices_exist() const
     }
     return false;
 }
+
+void GraphicsManagement::set_console(Graphics::Console& console)
+{
+    m_console = console;
+
+    if (auto* boot_console = boot_framebuffer_console.exchange(nullptr)) {
+        // Disable the initial boot framebuffer console permanently
+        boot_console->disable();
+        // TODO: Even though we swapped the pointer and disabled the console
+        // we technically can't safely destroy it as other CPUs might still
+        // try to use it. Once we an, drop the leaked ref when we created it.
```

Comment:
`Once we an, drop the leaked ref when we created it.`
I don't understand this sentence, can you clarify on this?

Reply:
Hm yeah lol don't know what I fabricated there...